src/board_controller/openbci/ganglion_native.cpp

```diff
@@ -442,10 +444,9 @@ void GanglionNative::adapter_1_on_scan_found (
 
     if (found)
     {
-        {
-            std::lock_guard<std::mutex> lk (m);
-            ganglion_peripheral = peripheral;
-        }
+        firmware = strncmp (peripheral_identified, "Ganglion 1.3", 12) == 0 ? 3 : 2;
```

Comment:
dont remember right now, but more likely there was a reason why I added extra scope with { and } for lock guard. Maybe I wanted this to be released before calling `notify_one` and maybe its more correct...  lets return it